README.md

```diff
@@ -1 +1,50 @@
-# Hello-world
\ No newline at end of file
+# Hello world
+
+In this project, you will set up a "Hello world" repository. No complex coding is required for this exercise. Your goal here is to master all of the tools and best practices you learned about in previous steps. You will be using them in all Microverse projects and most likely in your future job as well, so it is important to know them!
+
+## Built With
+
+- Html
+- Css
+- Linters
+
+
+## Getting Started
+
+**This is an example of how you may give instructions on setting up your project locally.**
+**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+### Setup
+
+### Install
+
+### Usage
+
+### Run tests
+
+### Deployment
```

Comment:
Since you did not 
- [ ] install any third-party libraries or dependencies, 
- [ ] Use any testing dependency.
- [ ] Deploy the project.
`You can remove those fields. ` What do you think?